lib/astarte/client/appengine/stats.ex

```diff
@@ -17,8 +17,22 @@
 #
 
 defmodule Astarte.Client.AppEngine.Stats do
+  @moduledoc """
+  Module to retrieve devices stats
```

Comment:
Ed. note: I lack some context here. If the **plurality** of devices is to be highlighted, I'd say:

Module to retrieve stats **from devices**.

Otherwise,

Module to retrieve **device** stats.

works for both a single device and multiple devices.